tests/remote/manifest.ini

```diff
@@ -1,3 +1,2 @@
-[test_fennec.py]
```

Comment:
Please not just remove this test but instead rename it so that we can have remote tests to ensure that real downloads pass as well.